### ISSUE TITLE
If content is not clickable . How can we close drawer while clicking content. I want both content not clickable and close drawer when content is clicked :D 

### ISSUE THREAD

**Issue by XeRrIc** [yarolegovich/SlidingRootNav#40]

**Comment by mccccccmike** [comment]
please help,i have the same issue

**Comment by skliba** [comment]
@yarolegovich yeah, need the same behaviour implemented if possible. Would greately appreciate it.

To clarify, when the drawer is opened we don't want the content to fetch the clicks, we just want the click to close the drawer back. Like a normal navigation drawer behaviour. 
Is that possible?

**Comment by surindersinghbindra** [comment]
add these lines to 
`newRoot.setOnTouchListener(new View.OnTouchListener() {
            @Override
            public boolean onTouch(View view, MotionEvent motionEvent) {
                if (motionEvent.getAction() == MotionEvent.ACTION_DOWN) {
                    newRoot.closeMenu();
                }
                return true;
            }
        });`
[https://github.com/surindersinghbindra/KampFire/blob/master/app/src/main/java/com/yarolegovich/slidingrootnav/SlidingRootNavBuilder.java](url)
line number 191 and make 
.withContentClickableWhenMenuOpened(false)

**Comment by riocallos** [comment]
Use this instead. The previous fix disabled the sliding gesture from the left side of the screen to open menu.

newRoot.setOnTouchListener(
            new View.OnTouchListener() {
                @Override public boolean onTouch(View view, MotionEvent motionEvent) {
                    if (motionEvent.getAction() == MotionEvent.ACTION_DOWN) {
                        newRoot.closeMenu();
                        return true;
                    }
                    return false;
                }
            });

**Comment by ShanmugarajGitHub** [comment]
Close side drawer when pressing out side
Added below line in MainActivity : 

override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        var slidingRootNav: SlidingRootNav = = SlidingRootNavBuilder(this)
                .withToolbarMenuToggle(toolbar)
                .withMenuOpened(false)
                .withContentClickableWhenMenuOpened(false)
                .withSavedState(savedInstanceState)
                .withMenuLayout(R.layout.menu_left_drawer)
                .inject()
}
override fun dispatchTouchEvent(ev: MotionEvent): Boolean {
        slidingRootNav!!.closeMenu() //
        return super.dispatchTouchEvent(ev)
}
